tensorflow/c/kernels_test.cc

```diff
@@ -73,6 +73,12 @@ static void* MyCreateFunc(TF_OpKernelConstruction* ctx) {
   EXPECT_EQ(TF_FLOAT, type);
   TF_DeleteStatus(status);
 
+  // Exercise kernel NodeDef name read 
+  string_view name_string_view = TF_OpKernelConstruction_GetName(ctx); 
+  const char* kernel_name = "SomeKernelName";
+  const char* candidate_kernel_name = std::string(name_string_view.data, 
+                                                  name_string_view.len).c_str(); 
+  EXPECT_EQ(0, strcmp(kernel_name, candidate_kernel_name)); 
```

Comment:
EXPECT_EQ should automatically handle std::string equality; let's keep kernel_name and candidate_kernel_name as std::string.